wiki/Community/Project_Loved/en.md

```diff
@@ -32,11 +32,13 @@ Although many [Pending and Work-in-progress](/wiki/Beatmap/Category#work-in-prog
 
 Beatmaps entering the Loved category are required to follow some additional rule sets:
 
-- [The Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+- [Partially the Metadata section of the Ranking Criteria](/wiki/Ranking_Criteria#metadata)
+  - Song title, artist and source should be correct in all cases.
+  - Beatmap tags should properly list all contributors (guest difficulty creators, storyboarders, etc.).
 - [Song Content Rules](/wiki/Rules/Song_Content_Rules)
 - [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations)
 
-Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap's audio files should be from the highest quality source available while having an average bit rate no greater than 192kbps.
+Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap's audio files should be at least 128kbps mp3 equivalent.
```

Comment:
slightly assuming on what "mp3 equivalent" is supposed to mean

```suggestion
Additionally, in order to keep the audio quality of the song at a reasonable level, beatmap audio files should have an average bitrate of at least 128kbps.
```

Reply:
"128 kbps" wording here sound like the audio should be in loved section, when in order ranking criteria, 192 kbps is the higher audio can be used.

Reply:
we always allow bloated audios to be loved as-is, and allow ones that are above 192kbps to be loved under certain conditions, so including the audio RC here would be misleading. 
our only requirement is for the audio to be at least 128kbps.